README.md

```diff
@@ -23,6 +23,25 @@ computers.
   </a>
 </div>
 
+## Example
+
+```js
+const start = new Date();
+const res = await fetch("https://ipfs.io/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi/");
+console.log(`TTFB: ${new Date() - start}ms`);
+```
+
+```bash
+$ zinnia run example.js
+TTFB: 235ms
+```
+
+## Installation
+
+```bash
+$ cargo install zinnia
+```
```

Comment:
This is going to change soon, see https://github.com/filecoin-station/zinnia/pull/48/commits/27a836fd99e7b2325e5a12bff0c483562fa237cb

To avoid duplication and out-of-date content, I propose keeping this section but replacing the content with a link to CLI's README.

Reply:
In the future, I'd like to provide pre-built binaries to simplify the installation instructions.
- https://github.com/filecoin-station/zinnia/issues/23